src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.Windows.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.UI.Xaml;
+
+namespace Microsoft.Maui.Controls
+{
+	public partial class VisualElement
+	{
+		public static void MapAccessKeyHorizontalOffset(IViewHandler handler, View view) =>
+			Platform.VisualElementExtensions.UpdateAccessKey((FrameworkElement)handler.PlatformView, view);
+		
+		public static void MapAccessKeyPlacement(IViewHandler handler, View view) =>
+			Platform.VisualElementExtensions.UpdateAccessKey((FrameworkElement)handler.PlatformView, view);
+		
+		public static void MapAccessKey(IViewHandler handler, View view) =>
```

Comment:
What is this AccessKey and how can I test it?

Reply:
By pressing Alt on the page (which was not added previously)